packages/sdk/src/client/backend.ts

```diff
@@ -179,7 +180,31 @@ export class MetromApiClient {
                 `response not ok while fetching activity for address ${params.address}: ${await response.text()}`,
             );
 
-        return (await response.json()) as Activity[];
+        const activities = (await response.json()) as BackendActivity[];
+
+        return activities.map((activity) => {
+            if (activity.payload.type === "claimReward") {
+                const processedToken = processErc20Token(
+                    activity.payload.token,
+                );
+                const rawAmount = BigInt(activity.payload.amount);
+
+                return {
+                    ...activity,
+                    payload: {
+                        ...activity.payload,
+                        amount: {
+                            raw: activity.payload.amount,
```

Comment:
Shouldn't this be `rawAmount`? The `raw` attribute is supposed to be a `bigint`, while the other a `number`

Reply:
Fixed in b735f26b5dfbc13b3a57dba3b75733a8427e98ca.